packages/tracing-api/src/services/bucketService.ts

```diff
@@ -10,13 +10,16 @@ export const bucketServiceBuilder = (s3Client: S3Client) => {
     async writeObject(
       file: ExpressMulterFile,
       bucketS3Key: string,
+      isReplacing?: boolean,
```

Comment:
let's call this `useReplacemenetBucket`, it's a bit more clear.

We will remove the replacement bucket anyway after the PoC